.changes/unreleased/Features-20230303-144700.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Features
+body: Make MAXIMUM_SEED_SIZE configurable
+time: 2023-03-03T14:47:00.079887-05:00
+custom:
+  Author: acurtis-evi
+  Issue: "7117"
```

Comment:
I guess these two entries could remain separate and still link to the two separate issues. One feature change and one under the hood? You probably already thought of that. We can update the PR to resolve both issues as well.

Reply:
Maybe we could merge our changies once this is done (changie new allows multiple usernames, mine is acurtis-evi)

✗ GitHub Username(s) (separated by a single space if multiple): █